### PR TITLE
Nightwatch tests

### PR DESCRIPTION
**Pull request by adgad**
/cc @matthew-andrews @spriggsha

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ test:
 	# Run all tests except for smoke tests
 	$(MAKE) build-production
 	export ELASTIC_SEARCH_URL=http://123.foundcluster.com:9200/v1_api/item; export HOSTEDGRAPHITE_APIKEY=123; export ENVIRONMENT=production; mocha --reporter spec tests/server/
-
+	./node_modules/nightwatch/bin/nightwatch --test tests/browser/tests/* --config ./tests/browser/nightwatch.json -e ie10,firefox,chrome
```
Comment (matthew-andrews) [inline Makefile]: think this should be changed to ideally `nightwatch --test…` or at least `./node_moduls/.bin/nightwatch test…`

File: tests/browser/nightwatch.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  
+  "test_settings" : {
+    "default" : {
+      "launch_url" : "${TEST_URL}",
+      "selenium_port"  : 80,
+      "selenium_host"  : "ondemand.saucelabs.com",
+      "silent": true,
+      "screenshots" : {
+        "enabled" : true,
+        "path" : "./screenshots"
```
Comment (matthew-andrews) [inline tests/browser/nightwatch.json]: are we actually using the screenshots?

Comment (matthew-andrews) [inline tests/browser/nightwatch.json]: ie. are they visible anywhere?

Comment (adgad) [inline tests/browser/nightwatch.json]: Good point, I can't see them anywhere (for passing or failing tests...)

Comment (matthew-andrews) [inline tests/browser/nightwatch.json]: i'll ask this on the docs PR

Comment (matthew-andrews) [inline tests/browser/nightwatch.json]: i'm fine with merging this in here now

Comment (matthew-andrews) [inline tests/browser/nightwatch.json]: ahh there is a note:-

https://github.com/Financial-Times/next/pull/8/files#diff-98e1507dbf0453862116156695a61717R84

**Pull request closed without merging**